### ISSUE TITLE
SQLite adapter fails to raise error

### ISSUE THREAD

**Issue by mattias-p** [zonemaster/zonemaster-backend#722]
When I configured Backend to use SQLite and did this:

```
$ zmb start_domain_test --domain example.com | jq .
{
  "jsonrpc": "2.0",
  "result": null,
  "id": 1
}
```

We should never be getting `"result": null` here. Something is wrong.

```
$ tail -n4 /home/mattiasp/.log/zonemaster/zm-rpcapi.log
DBD::SQLite::db selectrow_array failed: no such table: test_results at /home/mattiasp/zonemaster-backend/lib/Zonemaster/Backend/DB/SQLite.pm line 190.
DBD::SQLite::db do failed: no such table: test_results at /home/mattiasp/zonemaster-backend/lib/Zonemaster/Backend/DB/SQLite.pm line 223.
DBD::SQLite::db selectrow_array failed: no such table: test_results at /home/mattiasp/zonemaster-backend/lib/Zonemaster/Backend/DB/SQLite.pm line 226.
Use of uninitialized value $id in numeric gt (>) at /home/mattiasp/zonemaster-backend/lib/Zonemaster/Backend/DB/SQLite.pm line 228.
```

So I forgot to initialize my database. That's obviously wrong. But it should result in an error response. We're getting a successful response with `"result": null`. Also we're making three failed database queries. We should be aborting after the first failure.

**Comment by matsduf** [comment]
Will be resolved by #723

**Comment by matsduf** [comment]
Resolved by #723
